.env

```diff
@@ -1 +1,2 @@
-export RUSK_PROFILE_PATH=$PWD
\ No newline at end of file
+export RUSK_PROFILE_PATH=$PWD
+export RUSTFLAGS="--remap-path-prefix $HOME="
```

Comment:
We shouldn't impact the whole build process. Instead, we should use this variable only for the WASM commands. Example:

`contracts/transfer/Makefile`

from:
```
wasm: ## Build the WASM files
	@cargo rustc \
		--release \
		--target wasm32-unknown-unknown \
		-- -C link-args=-s
```

to:
```
wasm: ## Build the WASM files
        @RUSTFLAGS="--remap-path-prefix $$HOME=" \
                cargo rustc \
                --release \
                --target wasm32-unknown-unknown \
                -- -C link-args=-s
```